### DISCUSSION TITLE
Feb 5

### DISCUSSION THREAD

**Discussion by miniposi** [GymHubCommunity/GymHub-FE#103] [DailyMeeting]
### ⚽️ 회의 내용

* 개발 현황
  슬희: Oauth 하는 거 너무 귀찮 ~~
  수: 무한스크롤 이미 구현했는데 바꾸려니 화남
  수영: 게시글 상세보기 와이어프레임 시작해야지!!
  윤혁: 와이어프레임 올려둠 ~~ (할 거 없어서 ㅠ.ㅠ) -> 공통 부분도 같이 하기!!

* 전체 회의록

### 🏀 오늘 할 일

슬희: OAuth
수: 와이어 프레임
수영: 다른 거 와이어 프레임 ㅋㅋ
윤혁: 오류 고치고 남은 와이어 프레임

### 🎱 TMI

슬희: 마라샹궈 식당에서 소주랑 먹음 🤔
수: 피티 트레이너 너무 더럽다!! 환불 받으러 고고 ~
수영: 강남 가는 중 ... 🚃 결혼식 대관하러 가는...?!
윤혁: 에반게리온 ... 애니 조아 🤩

**Comment by hanseulhee** [comment]
환불 받아써여 ??
